### PR TITLE
submitting arviz

### PR DESCRIPTION
**Pull request by ericmjl**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

cc: @canyon289, @aloctavodia. @ColCarroll, @ahartikainen: would you all like to be included as maintainers of this recipe? It is quite lightweight, and basically involves approving the bot's auto-update once all tests pass. If you're not willing, not a problem: I will remove you from the list of maintainers here. No hard feelings, btw, we all will get busy at some point.

Also, do you all include the LICENSE file when uploading to PyPI. I think it is a good thing to do; LICENSE files are lightweight, and doing so attaches the license when distributing the software, and helps conda-forge's machinery ensure that the appropriate licenses are shared when uploaded to conda-forge's servers.

### CODE REVIEW

**Comment by ahartikainen** [comment]
Sounds fine.

Others can answer for the LICENCE questions.

**Comment by ColCarroll** [comment]
Thanks for getting this ready, @ericmjl! This is super helpful

- I'd love to be added as a maintainer
- We do include the LICENSE on pypi. It is in the MANIFEST (which is still inscrutable, but `check-manifest` is a wonderful package), and I just confirmed by downloading and unzipping the .tar.gz from pypi.

**Comment by canyon289** [comment]
I'd like to be included as a maintainer

**Comment by ericmjl** [comment]
@conda-forge/help-python, I'm not quite sure how to debug the current error showing up on Azure. Is there something I could do to rectify this issue?

**Comment by ahartikainen** [comment]
Is there a way to force Python 3 only?

**Comment by ericmjl** [comment]
AhhhH! That's why - I made it noarch Python, but that assumes 2.7 gets built as well, I think.

**Comment by ericmjl** [comment]
Ok, that clearly wasn't the issue here XD. My brain space is currently filled with pyjanitor dev stuff at PyCon, so I think I will have to defer to the experts on this issue.

**Comment by aloctavodia** [comment]
Thanks @ericmjl for helping with this. I'd also like to be included as a maintainer :-)

File: recipes/arviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a1967b56d7bf1ee4464788455c1a4fa0e2f994aef7d49c3b3dcc981f4f376727
+build:
+  skip: true  # [py2k]
```
Comment (isuruf) [inline recipes/arviz/meta.yaml]:
```suggestion
  noarch: python
```

File: recipes/arviz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python
```
Comment (isuruf) [inline recipes/arviz/meta.yaml]:
```suggestion
    - python >=3
```

File: recipes/arviz/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    - pip
+  run:
+    - python
```
Comment (isuruf) [inline recipes/arviz/meta.yaml]:
```suggestion
    - python >=3
```

File: recipes/arviz/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - matplotlib>=3.0
```
Comment (isuruf) [inline recipes/arviz/meta.yaml]:
```suggestion
    - matplotlib >=3.0
```

**Comment by isuruf** [comment]
Looks like the source tarball in PyPI doesn't have requirements.txt and setup.py tries to open that file.

**Comment by ericmjl** [comment]
@ColCarroll thanks for updating the MANIFEST! You might also need to cut a new patch release for conda-forge to pull.

**Comment by ericmjl** [comment]
Woohoo! This looks like it's all passing! cc: @isuruf

File: recipes/arviz/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    - xarray
+    - netcdf4
+
+test:
+  imports:
+    - arviz
+
+about:
+  home: https://github.com/arviz-devs/arviz
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
```
Comment (chrisburr) [inline recipes/arviz/meta.yaml]: Can you drop these comments from the example

File: recipes/arviz/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Exploratory analysis of Bayesian models with Python'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (chrisburr) [inline recipes/arviz/meta.yaml]: And here

File: recipes/arviz/meta.yaml (hunk 5 above)
Comment (ericmjl) [reply recipes/arviz/meta.yaml]: Done!

File: recipes/arviz/meta.yaml (hunk 6 above)
Comment (ericmjl) [reply recipes/arviz/meta.yaml]: Done!

**Comment by ericmjl** [comment]
yayyy!!! Thank you to all, @chrisburr @ColCarroll @canyon289 @ahartikainen @aloctavodia @isuruf! :tada: :tada:

**Pull request merged**